src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -26,6 +28,26 @@
 from pants.rules.core.test import TestDebugRequest, TestOptions, TestResult, TestTarget
 
 
+DEFAULT_COVERAGE_CONFIG = dedent(f"""
+  [run]
+  branch = True
+  timid = False
+  relative_files = True
+  """)
+
+
+def get_coveragerc_input(coveragerc_content: str) -> InputFilesContent:
+  return InputFilesContent(
+    [
+      FileContent(
+        path='.coveragerc',
+        content=coveragerc_content.encode(),
+        is_executable=False,
+      ),
+    ]
+  )
+
+
```

Comment:
Bad rebase. This already exists.

Reply:
Also, can you move some of this coverage specific code to `coverage.py` and have the `python_test_runner.py` import it? Would be great to shrink `python_test_runner.py` a bit.